docs/bigquery-usage.rst

```diff
@@ -429,3 +443,112 @@ Poll until the job is complete:
    'done'
    >>> job.ended
    datetime.datetime(2015, 7, 23, 9, 30, 21, 334792, tzinfo=<UTC>)
+
+Exporting data (async)
+~~~~~~~~~~~~~~~~~~~~~~
+
+Start a job exporting a table's data asynchronously to a set of CSV files,
+located on GCloud Storage.  First, create the job locally:
```

Comment:
For non-code names, we should probably track the branding in the [official docs](https://cloud.google.com/storage/), which appears to be "Google Cloud Storage". 906e120 updates usage to match.